serialization/serialization_writer.go

```diff
@@ -72,4 +72,6 @@ type SerializationWriter interface {
 	GetSerializedContent() ([]byte, error)
 	// WriteAdditionalData writes additional data to underlying the byte array.
 	WriteAdditionalData(value map[string]interface{}) error
+	// WriteAnyValues an object of unknown type as a json value
+	WriteAnyValues(key string, value interface{}) error
```

Comment:
shouldn't we make this one singular since we don't know for a fact will get something that holds multiple values?

Reply:
Yes, it should be `WriteAnyValue`